prac_05/emails.py

```diff
@@ -0,0 +1,27 @@
+def extract_name(email):
+    # Extract the name from the email by splitting at '@' and taking the first part
+    return email.split('@')[0].title()
+
+
+def main():
+    user_info = {}
```

Comment:
Dictionary naming conventians x_to_x, email_to_name looks like what you're after here.

Reply:
Thank you that has been adjusted